### PR TITLE
Added sort functionality (wikimedia hackathon contribution)

### PR DESCRIPTION
**Pull request by hks3333**
Added sort option above the table, now the data can be sorted on the basis of tool title/author. 

Updated README.

image

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -24,7 +24,40 @@ def index():
...
+def sortT():
+    session = Session()
+    curr_page = int(request.args.get("page",1))
+    tools = session.query(Tool).order_by(Tool.title).all()
+    paginated_tools=tools[(curr_page-1)*page_limit:curr_page*page_limit]
+
+    was_crawled = []
+    for tool in paginated_tools:
+        url_parsed = urlparse(tool.url)
+        if url_parsed.hostname != None and 'toolforge.org' in url_parsed.hostname :
```
Comment (gopavasanth) [inline app.py:39]: Modularise code, something similar to: 

def get_paginated_tools(session, order_by, curr_page):
    tools = session.query(Tool).order_by(order_by).all()
    total_pages = (len(tools) + page_limit - 1) // page_limit
    paginated_tools = tools[(curr_page - 1) * page_limit:curr_page * page_limit]
    
    was_crawled = [urlparse(tool.url).hostname and 'toolforge.org' in urlparse(tool.url).hostname for tool in paginated_tools]
    
    return paginated_tools, was_crawled, total_pages

@app.route("/sortT")
def sortT():
    session = Session()
    curr_page = int(request.args.get("page", 1))
    paginated_tools, was_crawled, total_pages = get_paginated_tools(session, Tool.title, curr_page)
    return render_template('index.html', tools=paginated_tools, was_crawled=was_crawled, curr_page=curr_page, total_pages=total_pages, route='sortT')

@app.route("/sortA")
def sortA():
    session = Session()
    curr_page = int(request.args.get("page", 1))
    paginated_tools, was_crawled, total_pages = get_paginated_tools(session, Tool.author, curr_page)
    return render_template('index.html', tools=paginated_tools, was_crawled=was_crawled, curr_page=curr_page, total_pages=total_pages, route='sortA')

**Comment by gopavasanth** [comment]
Also, it would be helpful to achieve sort behaviours of the table something similar to https://vuetifyjs.com/en/components/data-tables/basics/#usage

That means sort items based on click of table header.

**Comment by hks3333** [comment]
We are doing some modifications, you can expect a pr soon. Closing this pr for now.

**Comment by varun-s22** [comment]
Just a quick follow up here.
Can we please add sort by health as well 

cc @gopavasanth @hks3333

**Pull request closed without merging**